website/blog/2024-07-02-quickwit/index.mdx

```diff
@@ -0,0 +1,101 @@
+---
+slug: quickwit
+title: Log and trace management made easy. Quickwit Integration via Glasskube
+description: "A blog post on what Quickwit is and how it can help your data search efforts"
+authors: [ jpage ]
+tags: [ glasskube ]
```

Comment:
With the introduction of https://github.com/glasskube/glasskube/pull/874 all blog posts now need an image attribute as a preview image for the front page